templates/skeleton/app/routes/account.orders._index.tsx

```diff
@@ -164,7 +164,7 @@ export const CUSTOMER_FRAGMENT = `#graphql
       pageInfo {
         hasPreviousPage
         hasNextPage
-        hasNextPage
+        startCursor
```

Comment:
Thank you for finding this, but I think it needs both `hasNextPage` and `startCursor`